### PR TITLE
add de.lernsoftware_filius.Filius

### PR DESCRIPTION
**Pull request by mikebarkmin**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by mikebarkmin** [comment]
bot, build de.lernsoftware_filius.Filius

**Comment by mikebarkmin** [comment]
bot, build de.lernsoftware_filius.Filius

**Comment by mikebarkmin** [comment]
bot, build de.lernsoftware_filius.Filius

File: de.lernsoftware_filius.Filius.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          "sha256": "3334ec05947d31c864b72802d6af623628bb42a8dca60ced3c042fb27475053a",
+          "strip-components": 0
+        }
+      ]
+    },
+    {
+      "name": "filius-icon",
+      "buildsystem": "simple",
+      "build-commands": [
+        "for size in 16 32; do install -Dm644 filius${size}.png /app/share/icons/hicolor/${size}x${size}/apps/de.lernsoftware_filius.Filius.png; done"
```
Comment (hfiguiere) [inline de.lernsoftware_filius.Filius.json:44]: you are missing a 128x128 icon.

also the 64x64 icon isn't installed here.

File: filius [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+APPBASE="/app/filius"
+cd $APPBASE
+./filius.sh
```
Comment (hfiguiere) [inline filius:4]:
```suggestion
exec ./filius.sh
```

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (hfiguiere) [inline flathub.json:1]: you don't need this file if it's empty

File: de.lernsoftware_filius.Filius.json (hunk 1 above)
Comment (mikebarkmin) [reply de.lernsoftware_filius.Filius.json]: done

**Comment by mikebarkmin** [comment]
bot, build de.lernsoftware_filius.Filius

**Comment by mikebarkmin** [comment]
bot, build de.lernsoftware_filius.Filius

**Pull request closed without merging**